pallets/weights/src/pallet_identity.rs

```diff
@@ -176,38 +177,36 @@ impl pallet_identity::WeightInfo for WeightInfo {
     // Storage: Asset ScopeIdOf (r:1 w:0)
     // Storage: Asset BalanceOfAtScope (r:1 w:0)
     fn revoke_claim_by_index() -> Weight {
-        (88_241_000 as Weight)
+        (76_831_000 as Weight)
             .saturating_add(DbWeight::get().reads(5 as Weight))
             .saturating_add(DbWeight::get().writes(1 as Weight))
     }
     // Storage: Identity KeyRecords (r:2 w:1)
     fn set_secondary_key_permissions() -> Weight {
-        (53_854_000 as Weight)
+        (84_890_000 as Weight)
             .saturating_add(DbWeight::get().reads(2 as Weight))
             .saturating_add(DbWeight::get().writes(1 as Weight))
     }
-    fn permissions_cost(a: u32, p: u32, l: u32, e: u32) -> Weight {
+    fn permissions_cost(_a: u32, p: u32, l: u32, e: u32, ) -> Weight {
```

Comment:
These parameters are important and shouldn't be ignore.  The `a` parameter is for the number of asset permissions.

Reply:
Copied over:
```
            // Standard Error: 60_000
            .saturating_add((296_000 as Weight).saturating_mul(a as Weight))        
```

Reply:
This is still missing.